platform/features/conductor-redux/src/README.md

```diff
@@ -0,0 +1,187 @@
+## Notes
+API is notional for now, based on use-cases identified below. Possible the 
+use cases are not sufficient, so please include in comments 
+any other use cases you'd like to see. 
+
+Plan now is to start building out test suite for the use cases identified below
+in order to get the API functional. Need to discuss how UI aspects of timeline will be implemented.
+Propose in place refactoring of existing timeline rather than starting again.
+
+Some caveats / open questions
+* I don't understand the use case shown on page 52 of UI sketches. It shows RT/FT, with deltas, 
+with inner interval unlocked. Not sure what result would be, has inner end switched to fixed?
+Also example on page 55 in real-time where inner end < now. Is there a use case for this? Semantically, it's saying
+show me real time, but stale data. Why would a user want this? My feeling is that if the inner 
+OR outer ends are moved behind NOW in real-time mode then you drop into historical mode.
+* For the API itself, have ignored question of how it's namespaced / exposed. 
+Examples assume global namespace and availability from window object. 
+For now API implemented as standard standard Require JS AMDs. Could attach 
+to window from bundle.js. Perhaps attaching to window not best approach though...
+* Have not included validation (eg. start time < end time) or any other 
+business logic such as what happens when outer interval gets dragged 
+within range of inner interval. Focus is on teasing out the public API 
+right now. 
```

Comment:
Agree, validation should happen within the TimeConductor itself. Wanted to establish the functional API first, then think about validation concerns. There may be some value in exposing validation functions from the TimeConductor API to avoid duplicating validation in several places (in the TimeConductor class, as well as in the API).